types/pool.d.ts

```diff
@@ -18,5 +19,7 @@ declare namespace Pool {
     factory?(origin: URL, opts: object): Dispatcher;
     /** The max number of clients to create. `null` if no limit. Default `null`. */
     connections?: number | null;
+
+    interceptors?: { Pool?: DispatchInterceptor[] } & Client.Options["interceptors"]
```

Comment:
can you please add a test for these new types?

Reply:
I've added a few Jest tests to run some interceptors. Mainly in manipulating requests and responses.